release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -670,6 +670,11 @@ The following `oc` commands and flags for requesting tokens are now deprecated:
 
 Use the `oc create token` command instead to request tokens.
 
+[id="ocp-4-11-rhv-deprecations"]
+==== Red Hat Virtualization (RHV) in {product-title} will be deprecated
+
+Red Hat Virtualization (RHV) will be deprecated in an upcoming release of {product-title}. Additional details for when support for RHV will be removed from {product-title} will be announced in a future release.
```

Comment:
@janosdebugs we try to avoid giving any specific numbers, especially since it's one that is pretty far out. I kept the wording general to give room for things to change as we get closer. Thanks!